examples/gno.land/p/demo/grc/grc20/account_banker.gno

```diff
@@ -0,0 +1,70 @@
+package grc20
+
+import (
+	"std"
+)
+
+func PrevRealmBanker(b *Bank) *fnBanker {
```

Comment:
I've always disliked returning concrete but unexported types, and I'd rather this returned a `GRC20` interface.

Also, idea: just call this `RealmBanker`?

Reply:
i'll create interfaces instead, but probably not GRC20.